src/components/layouts/BaseLayout/BaseLayout.tsx

```diff
@@ -10,10 +9,7 @@ export default function BaseLayout(props: BaseLayoutProps): JSX.Element {
   return (
```

Comment:
React.FC does the same thing as the proposed interface and function. 
`const BaseLayout: React.FC<{}> = ({ children }) => {...`